services/main-frontend/src/pages/manage/organizations/[id]/components/DeleteOrganizationPopup.tsx

```diff
@@ -0,0 +1,73 @@
+import { css } from "@emotion/css"
+import React, { useState } from "react"
+import { useTranslation } from "react-i18next"
+
+import StandardDialog from "@/shared-module/common/components/dialogs/StandardDialog"
+
+interface Props {
+  show: boolean
+  setShow: React.Dispatch<React.SetStateAction<boolean>>
+  handleDelete: () => void
+}
+
+const DeleteOrganizationPopup: React.FC<Props> = ({ show, setShow, handleDelete }) => {
```

Comment:
We now have different types of confirmation dialogs. Should this be converted to be a new type of confirmation dialog? (since in this one you have to type something to be able to delete stuff). https://github.com/rage/secret-project-331/blob/c0cca55bd346df216ba0f6cd4b0ab8096208cae3/shared-module/packages/common/src/components/dialogs/DialogProvider.tsx#L169-L192

Also usually in these kind of dialogs usually I need to type something related to the thing I'm trying to delete, such as the organization name, to make sure I'm deleting the right thing.